velox/functions/prestosql/StringFunctions.h

```diff
@@ -202,6 +202,18 @@ struct HmacSha512Function {
   }
 };
 
+/// hmac_md5(varbinary) -> varbinary
+template <typename T>
+struct HmacMd5Function {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  template <typename TTo, typename TFrom>
```

Comment:
Since TTo and TFrom are always Varbinary, this template variable is not necessary. Please remove it.